numpy/typing/tests/data/reveal/ma.pyi

```diff
@@ -368,3 +382,243 @@ assert_type(MAR_2d_f4.T, np.ma.MaskedArray[tuple[int, int], np.dtype[np.float32]
 
 assert_type(MAR_2d_f4.nonzero(), tuple[_Array1D[np.intp], *tuple[_Array1D[np.intp], ...]])
 assert_type(MAR_2d_f4.nonzero()[0], _Array1D[np.intp])
+
+# Masked Array addition
```

Comment:
from here below it's all the maskedarray analogoue to the tests in `numpy/typing/tests/data/reveal/arithmetic.pyi`